Structure_oM/Results/Global Results/StoryDrift.cs

```diff
@@ -0,0 +1,64 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2024, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using System;
+using BH.oM.Base;
+using BH.oM.Analytical.Results;
+using BH.oM.Quantities.Attributes;
+using System.ComponentModel;
+
+
+namespace BH.oM.Structure.Results
+{
+    [Description("Story Drift for a given Load Case or Load Combination.")]
+    public class StoryDrift : StructuralGlobalResult, IImmutable, IResultItem
+    {
+        /***************************************************/
+        /**** Properties                                ****/
+        /***************************************************/
+
+        [Description("Story Name.")]
+        public virtual string Story { get; }
+
+        [Description("Drift Direction.")]
+        public virtual string Direction { get; }
+
+        [Ratio]
+        [Description("Drift Ratio.")]
```

Comment:
```suggestion
        [Description("Drift ratio.")]
```
This needs a bit more explanation - what is it the ratio of? Would it not be clearer if the StoreyDrift was a displacement?

Reply:
@peterjamesnugent,
thanks for this comment.
I add more details to the description. That's the ratio between the relative displacement of one floor to the other and the interstorey height at that floor ;)